### ISSUE TITLE
Enable all solr operators

### ISSUE THREAD

**Issue by davidread** [datagovuk/ckanext-dgu#311]
To do this there is a thing called 'solr_escape' which we can easily remove.

**Comment by rossjones** [comment]
Is this a duplicate of #306?

**Comment by davidread** [comment]
#306 is only concerned with the dash, but there are plenty of other operators in SOLR we can usefully enable, such as quoting and colon for per-field search. See also #308.

**Comment by rossjones** [comment]
We should blog about the changes once this is done.
